.repo/sample-list-generator/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "sample-list-generator",
```

Comment:
I think the relation between this script and the reo is not significant. If some dependencies are placed in the package.json at the root, it may cause some misunderstandings.